src/pow_c.c

```diff
@@ -190,7 +191,10 @@ static void *pworkThread(void *pitem)
         pworkInfo->n = -1;
     }
     pthread_mutex_unlock(pworkInfo->lock);
-    pthread_exit(NULL);
+}
+
+static void work_cb(uv_work_t *req) {
+    pworkThread(req->data);
```

Comment:
It is not necessary to keep function `pworkThread`. Instead, you can implement the existing logic in function `work_cb`.

Reply:
The function `pworkThread` has been removed.